### PR TITLE
extract codes from nodeStatusUpdater to UpdateNodeStatus() in pkg/util/node

### PR DESCRIPTION
**Pull request by NickrenREN**
```release-note
NONE
```

### CODE REVIEW

**Comment by NickrenREN** [comment]
@k8s-bot kops aws e2e test this

**Comment by NickrenREN** [comment]
@k8s-bot kops aws e2e test this

File: pkg/util/node/node.go [hunk 1]
```diff
@@ -183,3 +183,48 @@ func PatchNodeStatus(c clientset.Interface, nodeName types.NodeName, oldNode *v1
 	}
 	return updatedNode, nil
 }
+
+func UpdateNodeStatus(c clientset.Interface, node *v1.Node, attachedVolumes []v1.AttachedVolume) (error, bool) {
```
Comment (pmorie) [inline pkg/util/node/node.go]: There should be a few calls sites in the kubelet that can use this as well - check `kubelet_node_status.go`

**Review by pmorie — COMMENTED** [review]
Generally LGTM, just look for the remaining opportunities to de-dupe.

**Pull request closed without merging**